quickstart/nodejs/src/index.ts

```diff
@@ -0,0 +1,93 @@
+import { DefaultAzureCredential } from '@azure/identity';
+import { EntraIdCredentialsProviderFactory, REDIS_SCOPE_DEFAULT } from '@redis/entraid';
+import { createCluster } from '@redis/client';
+import * as net from 'node:net';
+
+const resourceEndpoint = process.env.AZURE_MANAGED_REDIS_HOST_NAME!;
+if (!resourceEndpoint) {
+    console.error('AZURE_MANAGED_REDIS_HOST_NAME is not set. It should look like: `cache-name.region-name.redis.azure.net`. Find the endpoint in the Azure portal. Do not include the port.');
+    process.exit(1);
+}
+// Azure Managed Redis default port
+const resourcePort = 10000
+
+let client;
+let endpointUrl = `rediss://${resourceEndpoint}:${resourcePort}`;
+console.log('Using Redis endpoint:', endpointUrl);
+
+try {
+
+    function getCluster() {

```

Comment:
Weird to define a function inside a try block. Can we eliminate the function and just run its contents inline to create the connection?

Reply:
I thought this is what you instructed when you didn't want the function outside the try/block.

Reply:
I was thinking the _contents_ of the method should go inside the try block. But you could define the function outside the try, and just make sure the call to it comes from within the try block